navigation/BottomTabNavigator.tsx

```diff
@@ -28,7 +28,7 @@ import { HeaderStyles } from "components/HeaderStyles";
 export type BottomTabParamList = {
   Companies: undefined;
   Maps: undefined;
-  Profile: undefined;
+  You: undefined;
```

Comment:
You would also need to change the name in the linking configuration to prevent the not found screen from showing when logging out.

Reply:
That explains alot. I forgot to test it for others than admin account. My bad. Will take a look and upload correction later

Reply:
-Where is this linking configuration you speak of?

Reply:
This
`Profile: {
            screens: {
              ProfileScreen: "profile",
              ProfileSwitchScreen: "profile/:screen/:id",
            },
          },
`
should be changed to this
`You: {
            screens: {
              ProfileScreen: "profile",
              ProfileSwitchScreen: "profile/:screen/:id",
            },
          },`
in the `LinkingConfigurations.ts` file